### DISCUSSION TITLE
ScreenToGif 2.40

### DISCUSSION THREAD

**Discussion by NickeManarin** [NickeManarin/ScreenToGif#1257] [Announcements]
# What's new?

• Small updates to theming, now the title bar follows the theme color as well.  
• Update to .NET 8 (Windows 10 1607 is the minimum supported version).  
• Updated Dutch, Chinese and Czech localization.  

### Fixed:

♦ Fix the collection modified error when opening a project.  
♦ Fixed the frame selection which was causing some frames to be pasted out of order.    
♦ Tweaked the border color around the recorder (for dark theme) as it was not matching the other colors.  

### More info:

[Known issues](https://github.com/NickeManarin/ScreenToGif/wiki/Help-▪-Known-issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/NickeManarin/ScreenToGif/releases/tag/2.40'>ScreenToGif 2.40</a>.</em>

**Comment by Dhyfer1** [comment]
I don't have .NET 8, instead I have .NET 3.1 and screentogif works fine, should I still update .NET?
